src/main/java/clara/test/facts/BeanTestFact.java

```diff
@@ -0,0 +1,38 @@
+package clara.test.facts;
+
+/**
+ * A Java Pojo for the express purpose of testing the behavior of compilation and execution of rules.
+ *
+ * This class should not be included in the released artifact, if it did make it into a released artifact it should be
```

Comment:
Why can't it be under `src/test/java` instead? ie, a dev-time-only `:source-paths`?

Reply:
Agreed, I don't understand why this needs to be in the main source path and then removed from the jar afterward rather than just put in a test source path from the start.

Reply:
My lein is not that strong, but yeah that works:
https://github.com/cerner/clara-rules/pull/449/commits/b41ffee72a58f19e94b2169f4e9b7ed35736551c